plugins/woocommerce-admin/client/products/fields/enriched-label.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button, Popover } from '@wordpress/components';
+import { Fragment, useState } from '@wordpress/element';
+import interpolateComponents from '@automattic/interpolate-components';
+import { Link } from '@woocommerce/components';
+import { recordEvent } from '@woocommerce/tracks';
+import { Icon, help } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import './enriched-label.scss';
+
+export default function EnrichedLabel( {
```

Comment:
Maybe we should remove the code in the `ProductTypeLabel` and `ProductType` components that this is based off of and re-use this component?
In that case I would move this component out of the `products/fields` directory and maybe in a `client/shared` directory instead.

Reply:
Good idea! I moved the component to `packages/js/componets` in commit 8e1ed6e. If it's ok with you, I'm going to add the changes to use that component in `ProductTypeLabel` and `ProductType` in a follow-up PR, to keep the scope of this PR smaller.